common/persistence/execution_manager_test.go

```diff
@@ -675,6 +675,100 @@ func TestDeserializeBufferedEvents(t *testing.T) {
 	}
 }
 
+func TestPutReplicationTaskToDLQ(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockedStore := NewMockExecutionStore(ctrl)
+	manager := NewExecutionManagerImpl(mockedStore, testlogger.New(t), nil)
+
+	now := time.Now().UTC().Round(time.Second)
```

Comment:
Nit: I'd use a fixed time like `now := time.Unix(123,456)` that way the test is completely reproduceable. 

I don't think it's super important, feel free to ignore :)